jdaviz/configs/specviz/helper.py

```diff
@@ -11,32 +11,56 @@ class SpecViz(ConfigHelper):
 
     _default_configuration = 'specviz'
 
-    def load_data(self, data, data_label='', format=None):
-        """Loads a data file or Spectrum1D object into SpecViz
+    def load_data(self, data, data_label=None, format=None):
+        """
+        Loads a data file or `~specutils.Spectrum1D` object into SpecViz.
 
-        :param data: Spectrum1D spectra, or path to compatible data file
-        :param data_label: Name/identifier of data
-        :param format: Spectrum1D data format to load
+        Parameters
+        ----------
+        data : str or `~specutils.Spectrum1D`
+            Spectrum1D spectra, or path to compatible data file.
+        data_label : str
+            The Glue data label found in the ``DataCollection``.
+        format : str
+            Loader format specification used to indicate data format in
+            `~specutils.Spectrum1D.read` io method.
         """
         # If no data label is assigned, give it a unique identifier
-        if not data_label:
+        if data_label is None:
             data_label = "specviz_data|" + uuid.uuid4().hex
+
         # If data provided is a path, try opening into a Spectrum1D object
         try:
             path = pathlib.Path(data)
+
             if path.is_file():
                 data = Spectrum1D.read(path, format=format)
             else:
                 raise FileNotFoundError("No such file: " + path)
         # If not, it must be a Spectrum1D object. Otherwise, it's unsupported
         except TypeError:
             if type(data) is SpectrumCollection:
-                raise TypeError("SpectrumCollection detected. Please provide a Spectrum1D")
+                raise TypeError("`SpectrumCollection` detected. Please "
+                                "provide a `Spectrum1D`.")
             elif type(data) is not Spectrum1D:
                 raise TypeError("Data is not a Spectrum1D object or compatible file")
+
         self.app.add_data(data, data_label)
         self.app.add_data_to_viewer('spectrum-viewer', data_label)
 
     def get_spectra(self):
         """Returns the current data loaded into the main viewer"""
         return self.app.get_data_from_viewer('spectrum-viewer')
+
+    def get_regions(self):
```

Comment:
Maybe this should be:
```suggestion
    def get_spectral_regions(self):
```
?  That makes a clearer distinction between this and what #159 is about.  And this part of the API is all about spectra so I think it will make user code more readable...